integration-tests/smoke/automation_test.go

```diff
@@ -104,12 +104,11 @@ func SetupAutomationBasic(t *testing.T, nodeUpgrade bool) {
 
 	registryVersions := map[string]ethereum.KeeperRegistryVersion{
 		"registry_2_0": ethereum.RegistryVersion_2_0,
-		//"registry_2_1": ethereum.RegistryVersion_2_1,
+		//"registry_2_1_conditional": ethereum.RegistryVersion_2_1,
+		//"registry_2_1_logtrigger": ethereum.RegistryVersion_2_1,
```

Comment:
can we add a comment here with a TODO that points to the ticket to enable these tests? I just don't want us to forget to turn them on lol

Reply:
https://smartcontract-it.atlassian.net/browse/AUTO-4617 will work on it tomorrow before Anirudh taking PTO :)